.github/workflows/actions.yml

```diff
@@ -0,0 +1,50 @@
+name: Google Fonts Flutter Unit Tests
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+jobs:
+  test:
+    name: Test on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-10.15]
+    steps:
+      # Set up Flutter and add it to the path.
+      - name: Clone Flutter repository with stable channel
+        uses: actions/checkout@v2
+        with:
+          repository: 'flutter/flutter'
+          ref: 'stable'
+          path: 'flutter'
+          fetch-depth: 0
+      - name: Add Flutter to the PATH for Unix
+        if: startsWith(matrix.os, 'macOS') || startsWith(matrix.os, 'ubuntu')
+        run: echo "$GITHUB_WORKSPACE/flutter/bin" >> $GITHUB_PATH
+      - name: Add Flutter to the PATH for Windows
+        if: startsWith(matrix.os, 'windows')
+        run: echo "${env:GITHUB_WORKSPACE}\flutter\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+      - name: Run Flutter doctor
+        run: flutter doctor -v
+
+      # Clone the google_fonts repository under `code`, to avoid conflicts with `flutter`.
+      - name: Clone the google_fonts repository
+        uses: actions/checkout@v2
+        with:
+          path: 'code'
+
+      # Run the tests for the repository.
```

Comment:
nit: Analyze, check format, and run tests for the repository

Reply:
Done.